scripts/common/utils.sh

```diff
@@ -8,11 +8,11 @@ check_vars() {
     local _var_names=("$@")
 
     for var_name in "${_var_names[@]}"; do
-        if [ -z "${!var_name}" ];then
+        if  [[ -v var_name ]]; then
```

Comment:
I believe it should be
```suggestion
        if  [[ -v $var_name ]]; then
```

Reply:
Done